### PR TITLE
Generate keys inside sshproxy

### PR DESCRIPTION
**Pull request by tomasfejfar**
Depends on https://github.com/keboola/db-extractor-common/pull/115

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,5 @@
 #VERSION 1.0.0
+FROM db-ex-common-sshproxy AS sshproxy
```
Comment (pivnicek) [inline Dockerfile]: I'm not sure this will work as per travis build error

File: generate-ssh-keys.sh [hunk 2]
```diff
@@ -7,7 +7,7 @@ PUBLIC_KEY_FILE="/tmp/ssh.key.pub"
 rm -f ${PRIVATE_KEY_FILE}
 rm -f ${PUBLIC_KEY_FILE}
 
-ssh-keygen -b 4096 -t rsa -f ${PRIVATE_KEY_FILE} -q -N '' && echo "Done"
+ssh-keygen -b 4096 -m PEM -t rsa -f ${PRIVATE_KEY_FILE} -q -N '' && echo "Done"
 
 export SSH_KEY_PRIVATE=$(cat ${PRIVATE_KEY_FILE}) && echo "Private key exported to SSH_KEY_PRIVATE"
```
Comment (pivnicek) [inline generate-ssh-keys.sh]: Can we not remove this file completely now?

File: Dockerfile (hunk 1 above)
Comment (tomasfejfar) [reply Dockerfile]: Travis was down, so I didn't bother to wait for it, sorry about that. I'll fix it. If you just run DC build, it correctly builds sshproxy first.

Comment (pivnicek) [reply Dockerfile]: aha, sorry!  I didn't realise

**Pull request merged**